how-to/operations/maintenance-mode.rst

```diff
@@ -0,0 +1,113 @@
+Maintenance mode
+================
+
+Overview
+--------
+
+Maintenance mode helps protecting the cluster from potentially disruptive maintenance operations. It is particularly useful when an operator needs to manually perform tasks on a node that may pose a risk of failure or data loss.
```

Comment:
protecting should either be 'helps protect' or 'helps by protecting'. The former is preferable here.

Additionally, you can optionally remove 'particularly' in the next sentence. I think it would be helpful to include an example as well, e.g.:

It is useful for performing maintenance tasks on a node that may result in a loss of data or disrupt running services such as firmware upgrades.

Reply:
Thanks for the suggestion. Updated in latest commit.